crates/full-node/sov-sequencer/src/preferred/replica_sync_task.rs

```diff
@@ -108,29 +108,14 @@ enum CompletedEvent {
 pub async fn spawn_replica_sync_task<S, Rt, Da>(
     sequencer: Arc<PreferredSequencer<S, Rt, Da>>,
     shutdown_receiver: watch::Receiver<()>,
-    latest_state_update: StateUpdateInfo<S::Storage>,
+    _latest_state_update: StateUpdateInfo<S::Storage>,
     latest_loaded_event_id: Option<u64>,
 ) -> JoinHandle<()>
 where
     S: Spec,
     Rt: Runtime<S>,
     Da: DaService<Spec = S::Da>,
 {
-    if let Err(e) = sequencer
```

Comment:
This code contains a bug, so instead of updating it, I removed it. A proper fix will be included as part of the replica workstream in upcoming PRs